addon/lib/StudyTelemetryCollector.jsm

```diff
@@ -33,47 +32,50 @@ XPCOMUtils.defineLazyModuleGetter(this, "PlacesUtils",
 XPCOMUtils.defineLazyServiceGetter(this, "Telemetry",
   "@mozilla.org/base/telemetry;1", "nsITelemetry");
 
+// unit-tested study helpers
+XPCOMUtils.defineLazyModuleGetter(
+  this, "Helpers", "resource://esper-pioneer-shield-study/lib/Helpers.jsm"
+);
+
 /**
  * Note: Setting studyUtils in this constructor due to above comments
  */
 class StudyTelemetryCollector {
```

Comment:
nit: I'm not sure this needs to be a class since you only ever create one instance of it and there's nothing that you *need* the constructor for. I would probably just rewrite this to a simple object much like `Pioneer` in `Pioneer.jsm`.

Reply:
It doesn't, but since it was a class when the add-on passed qa and the risk for bugs increase the more lines of code is changed / methods refactored, I chose to leave it as it is. For the next add-on, I will use a simple object.